### ISSUE TITLE
new opportunities submitted, no email....

### ISSUE THREAD

**Issue by ultrasaurus** [18F/midas-open-opportunities#75]
It seems like maybe notifcation is only sent when it goes from draft -> open, not when originally posted as a draft...

![image](https://cloud.githubusercontent.com/assets/41906/6136415/539e1ca8-b125-11e4-9ed2-b558dbadfbc9.png)

... Lisa thinks she has gotten draft state notifications before, so maybe this was a special case

**Comment by dhcole** [comment]
I don't see any errors in the logs or in MainDrill (the mailing API). I do see successful sends over the last few days. 

![screen shot 2015-02-17 at 5 58 39 pm](https://cloud.githubusercontent.com/assets/170641/6239142/f1a27c9c-b6ce-11e4-8fce-b507f4039958.png)

It's possible this is related to the missing username issue that's addressed by https://github.com/18F/midas/pull/591

I tested locally too and can't recreate the issue. Closing until we see it again.
